clients/spark/build.sbt

```diff
@@ -174,7 +174,7 @@ ThisBuild / developers := List(
 )
 
 ThisBuild / versionScheme := Some("early-semver")
-ThisBuild / organization := "io.treeverse"
```

Comment:
Should we clean up old releases with the previous organization?

Reply:
Will do it in a month or so, after people switch to the new one